source/user/gpu.rst

```diff
@@ -124,75 +117,36 @@ orbitals followed by orbital localization using the Boys method on the CPU::
     mf = mf.to_cpu()
     loc_orb = lo.Boys(mol, mf.mo_coeff[:,[2,3,4]]).kernel()
 
-**GPU Implementation Availability**: The `to_gpu` method is implemented for
+**GPU Implementation Availability**: The :func:`to_gpu` method is implemented for
 almost all methods in PySCF. However, the actual availability of GPU4PySCF
 implementations for specific modules may vary. If a GPU4PySCF module is
-available, `to_gpu` will return a GPU4PySCF instance. Otherwise, it will raise a
-`NotImplementedError`.
+available, :func:`to_gpu` will return a GPU4PySCF instance. Otherwise, it will raise a
+:func:`NotImplementedError`.
 
 Functionalities supported by GPU4PySCF
 ======================================
-.. list-table::
-   :widths: 25 25 25 25
-   :header-rows: 1
-
-  * - Method
-    - SCF
-    - Gradient
-    - Hessian
-  * - direct SCF
-    - O
-    - GPU
-    - CPU
-  * - density fitting
-    - O
-    - O
-    - O
-  * - LDA
-    - O
-    - O
-    - O
-  * - GGA
-    - O
-    - O
-    - O
-  * - mGGA
-    - O
-    - O
-    - O
-  * - hybrid
-    - O
-    - O
-    - O
-  * - unrestricted
-    - O
-    - O
-    - O
-  * - PCM solvent
-    - GPU
-    - GPU
-    - FD
-  * - SMD solvent
-    - GPU
-    - GPU
-    - FD
-  * - dispersion correction
-    - CPU*
-    - CPU*
-    - FD
-  * - nonlocal correlation
-    - O
-    - O
-    - NA
-  * - ECP
-    - CPU
-    - CPU
-    - CPU
-  * - MP2
-    - GPU
-    - CPU
-    - CPU
-  * - CCSD
-    - GPU
-    - CPU
-    - NA
+
+====================== ===== ========= =========
+Method                 SCF   Gradient  Hessian
+direct SCF             O     GPU       CPU
+density fitting        O     O         O
+LDA                    O     O         O
+GGA                    O     O         O
+mGGA                   O     O         O
+hybrid                 O     O         O
+unrestricted           O     O         O
+PCM solvent            GPU   GPU       FD
+SMD solvent            GPU   GPU       FD
+dispersion correction  CPU*  CPU*      FD
+nonlocal correlation   O     O         NA
+ECP                    CPU   CPU       CPU
+MP2                    GPU   CPU       CPU
+CCSD                   GPU   CPU       NA
+====================== ===== ========= =========
+
+- ‘O’: carefully optimized for GPU. 
+- ‘CPU’: only cpu implementation. 
+- ‘GPU’: drop-in replacement or na¨ıveimplementation. 
```

Comment:
Change "na¨ıveimplementation" to "naive implementation".

Reply:
Done